packages/react-components/react-search/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -62,6 +62,12 @@ const useContentAfterStyles = makeStyles({
     paddingLeft: tokens.spacingHorizontalM,
     columnGap: tokens.spacingHorizontalXS,
   },
+  focused: {
+    display: 'flex',
+  },
```

Comment:
do you need to explicitly apply `display: flex` when focused? If not, I'd lean towards just conditionally applying the `unfocused` styles